### PR TITLE
Remove need to preinstall operator-ui

### PR DESCRIPTION
**Pull request by HenryNguyen5**
This cleans up our CI pipeline from the changes made in https://github.com/smartcontractkit/chainlink/pull/7697

### CODE REVIEW

File: core/web/middleware.go [hunk 1]
```diff
@@ -21,10 +21,8 @@ import (
 // inside this module. To achieve this, we direct webpack to output all of the compiled assets
 // in this module's folder under the "assets" directory.
 
-// HINT: if you are coming here because of this error:
-// `web/middleware.go:28:12: pattern assets: no matching files found`
-// then you must first run `make operator-ui` in the root dir to fetch the frontend
-//nolint
+// nolint
+//
```
Comment (jmank88) [inline core/web/middleware.go:25]: Might be able to just drop this.
```suggestion
```

**Comment by HenryNguyen5** [comment]
Seems like one of the action files has a ton of whitespace changes since it was improperly formatted before, github lets you hide whitespace changes to see the real changes easier

**Review by jmank88 — APPROVED** [review]

File: core/web/middleware.go (hunk 1 above)
Comment (jmank88) [inline core/web/middleware.go:25]: Was this nolint comment still required? (and is this extra line added by gofmt?) 
We'd like to clean them to be proper machine readable comments, like: `//nolint:<linter>` (no leading space)

Comment (jmank88) [reply core/web/middleware.go:25]: I don't know which specific rule this is for though

**Review by jmank88 — APPROVED** [review]

**Review by chainchad — APPROVED** [review]

**Pull request merged**